README.md

```diff
@@ -27,7 +27,55 @@ See the [documentation](https://gradlex.org/jvm-dependency-conflict-resolution/)
 Note: There are rules in this plugin (in particular for _Guava_) which work _better_ with Gradle `7+`.
 This is because Gradle 7 added support for the `org.gradle.jvm.environment` attribute that allows Gradle to better distinguish between _standard-jvm_ and _android_ projects.
 
-### Notes on the Plugins history
+## Contributing new capability rules
```

Comment:
If you put this into CONTRIBUTING.md, I think new contributors will be notified by GitHub to review that file.